layouts/events/card.html

```diff
@@ -2,15 +2,17 @@
 <div class="col">
   <div class="card mb-4 shadow-sm h-md-250">
     {{ if isset .Params "images" }}
-    {{ if gt (len .Params.images) 0 }}<img class="card-img-top" src="{{ (index .Params.images 0) }}">{{ end }}
+    {{ if gt (len .Params.images) 0 }}<img class="card-img-top" src="{{ (index .Params.images 0) }}" />{{ end }} 
     {{ end }}
     <div class="card-body">
       <h5 class="card-title">
         <a class="card-link" href="{{ .RelPermalink }}">{{ .Title }}</a>
       </h5>
-      <p class="card-text mb-1 text-muted">{{ dateFormat "Mon Jan 2, 2006" $startDate }}</p>
+    </div>
+    <div class="card-footer">
+      <small class="card-text mb-1 text-muted">{{ dateFormat "Mon Jan 2, 2006" $startDate }}</small>
       {{ with .Params.location }}
-      <p class="card-text small mb-auto">{{ .name }}</p>
+      <small class="card-text small mb-auto">{{ .name }}</small>
       {{ end }}
```

Comment:
I am thinking maybe we should keep the design with the date in a <p/> element?

Reply:
The small is still quite readable to me + there might be a clearer hierarchy of information, but as the font-size difference is 2px, either one looks alright!

Reply:
Yea, either one looks fine but I guess it might make it slightly harder for a small proportion of our viewers to read it without looking closely?